.gitlab-ci.yml

```diff
@@ -45,27 +45,37 @@ cache dependencies:
     | xargs nix-store --query --requisites
     | cachix push jasmin'
 
-coq:
+coq-program:
   stage: prove
   variables:
     EXTRA_NIX_ARGUMENTS: --arg coqDeps true
   extends: .common
   script:
-  - nix-shell --arg inCI true $EXTRA_NIX_ARGUMENTS --run 'make -j$NIX_BUILD_CORES -C proofs'
+  - nix-shell --arg inCI true $EXTRA_NIX_ARGUMENTS --run 'make -j$NIX_BUILD_CORES -C proofs Makefile.coq'
+  - nix-shell --arg inCI true $EXTRA_NIX_ARGUMENTS --run 'make -j$NIX_BUILD_CORES -C proofs -f Makefile.coq compiler/jasmin_compiler.vo'
   - nix-shell --arg inCI true $EXTRA_NIX_ARGUMENTS --run 'make -j$NIX_BUILD_CORES -C compiler CIL'
```

Comment:
I was convinced that you cannot just `make CIL` in `compiler` in a fresh clone, I thought that it was needed to do something in the `proofs` folder before, but after testing it is not true. Could we just `make -C compiler CIL`? Maybe the goal is to test separately the compilation of the Coq files and the extraction?

Reply:
OMG, you’re right. Do you think the `jasmin_compiler` coq module should be kept?

Reply:
Thanks. I’ve got the same feeling. Let me try the simplification you’ve just suggested and see what happens.

Reply:
Alright. CI confirms your findings. Thanks.